include/af/algorithm.h

```diff
@@ -306,11 +306,11 @@ namespace af
     template<typename T> void max(T *val, unsigned *idx, const array &in);
 
     /**
-       C++ Interface inclusive sum (cumulative sum) of an array
+       C++ Interface for computing the cumulative sum (inclusive) of an array
 
        \param[in] in is the input array
-       \param[in] dim The dimension along which exclusive sum is performed
-       \return the output containing exclusive sums of the input
+       \param[in] dim is the dimension along which inclusive sum is performed
```

Comment:
Similar to my first comment, `perform` is mostly with subjective contexts. Perhaps `calculated` or `computed`.